project/plugins.sbt

```diff
@@ -3,9 +3,7 @@ logLevel := Level.Warn
 
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.0")
 
-addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
```

Comment:
Development of this plugin has been discontinued.
The plugin has old conflicting dependencies with scalablytyped.